dev/com.ibm.ws.microprofile.reactive.messaging_fat/publish/servers/CheckpointSimpleRxMessagingServer/bootstrap.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2017, 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+osgi.console=5471
+com.ibm.ws.logging.trace.specification = *=info:logservice=detail:REACTIVEMESSAGE=all:com.ibm.ws.microprofile.reactive.messaging.kafka.adapter.impl.*=FINEST:org.apache.kafka.*=all
+websphere.java.security.exempt=true
+io.openliberty.checkpoint.allowed.features=localConnector-1.0
```

Comment:
`localConnector-1.0` is now enabled for InstantOn.  You don't need this line anymore.